lib/fog/linode/models/compute/kernels.rb

```diff
@@ -11,20 +11,27 @@ def all
           load kernels
         end
 
+        def all_kvm
+          load kernels(:isKVM => 1)
+        end
+
+        def all_xen
+          load kernels(:isXen => 1)
+        end
```

Comment:
@plribeiro3000 I wasn't sure about this either. I'm new to the fog project, so I'm not sure what the usual convention would be. I'm happy to change it per your suggestion if you think that would be better.